src/test/pivot_root.c

```diff
@@ -0,0 +1,41 @@
+/* -*- Mode: C; tab-width: 8; c-basic-offset: 2; indent-tabs-mode: nil; -*- */
+
+#include "nsutils.h"
+#include "rrutil.h"
+
+int main(void) {
+  try_setup_ns(CLONE_NEWNS);
```

Comment:
Shouldn't we print EXIT-SUCCESS here and exit if this fails?

Alternatively, `try_setup_ns` could just do that itself after it prints the `Skipping test` warning.